tests/Fixtures/Integration/priority/no_superfluous_elseif,simplified_if_return.test

```diff
@@ -0,0 +1,22 @@
+--TEST--
```

Comment:
If the fixer supports fixing conditions with `else` or `elseif`, I think there is no priority issue with the `no_useless_else` and `no_superfluous_elseif` fixers.

Reply:
* `no_superfluous_elseif`: you are right, but I would prefer to keep the integration test to explicitly tell what I expect from the two combined
* `no_useless_else`: it would be much more complex to handle in this fixer the `if`+`else` condition rather than let `no_useless_else` simplify it first; morover, as for `no_superfluous_elseif`, I would prefer to keep the integration test to explicitly tell what I expect from the two combined

Reply:
if there is no priority issue but we simply want to ensure both of them are playing nicely together (regardless of executing order), those tests shall be moved from `priority` to `misc` folder

Reply:
The priority dependency with `no_useless_else` and `no_superfluous_elseif` still exists, right? IMO it's fine for now. If we want to make the fixer more autonomous, that can happen later.